challenges/2021-09-20-tic-tac-toe-winner/solutions/ruby/brehberg/brehberg.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+POSSIBLE_ROWS = [
+    [1,2,3],
+    [4,5,6],
+    [7,8,9],
+    [1,4,7],
+    [2,5,8],
+    [3,6,9],
+    [1,5,9],
+    [3,5,7]
+  ].freeze
+  
+  def who_wins(board)
+    state = []
+    
+    # populate the board state
+    board.each.with_index(1) do |move, index|
+      row, col = move
+      state[row*3+col+1] = index.odd? ? 'A' : 'B'
```

Comment:
I love the manual handling of 2d indexing to 1d.

Reply:
It was much easier for me to enter the possible winning rows that way. Starting with index 1 makes my brain happier. 😅